files/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: username
+  password: you_password
```

Comment:
@MProG what do you think about skipping password for development db. it only makes development harder